.github/workflows/ci.yml

```diff
@@ -85,9 +85,6 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Build Docker image
-        uses: ./.docker/lint-xml-configuration
```

Comment:
@Slamdunk 

This step is here only to separate building of the Docker image from using it.